plugins/foreman_statistics/1.0/index.md

```diff
@@ -6,17 +6,18 @@ version: 1.0
 #robots: noindex
 ---
 
-<div class="bs-callout-warning bs-callout">
-  <h4>This is a Work in progress</h4>
-  Plugin will be available only in the next major release!
-</div>
-
 # 1. {{ page.title }}
 
 This plugin enables Foreman to show a nice statistics dashboard and create Trend charts.
 
+## 1.1 Release notes
+
+### Drop the cron job
+
+Foreman statistics 1.0 onwards automates data collection by facilitating a dynflow processing engine and no manual steps are required.
+If you've set a cron job to execute `foreman-rake trends:counter` manually in the past, please remove it.
```

Comment:
Previously it was shipped in foreman itself. If users modified the cronjob, a yum update will not update it either. So even if they didn't modify the trends counter explicitly, they still need to look. So I'd phrase it a bit more open.
```suggestion
Users upgrading from older versions should ensure to remove any cronjobs that call `foreman-rake trends:counter`.
```